src/app/component/eco-news/components/remaining-count/remaining-count.component.html

```diff
@@ -1,2 +1,2 @@
-<p *ngIf="[remainingCount] === 1">{{ remainingCount }} item found </p>
-<p *ngIf="[remainingCount] !== 1">{{ remainingCount }} items found</p>
+<p *ngIf="remainingCount > 0">{{ remainingCount }} item found </p>
+<p *ngIf="remainingCount < 1">{{ remainingCount }} items found</p>
```

Comment:
it's not correspond to the flow what was
explain this

Reply:
fixed